frontend/src/components/admin/AdminVerifiedMinesList.js

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import { Table } from "antd";
+import PropTypes from "prop-types";
+import CustomPropTypes from "@/customPropTypes";
+import * as router from "@/constants/routes";
+import { Link } from "react-router-dom";
+import NullScreen from "@/components/common/NullScreen";
+import { formatDate } from "@/utils/helpers";
+
+/**
+ * @class AdminDashboard houses everything related to admin tasks, this is a permission-based route.
+ */
+
+const propTypes = {
+  minesVerifiedStatusList: PropTypes.arrayOf(CustomPropTypes.mineVerificationStatus).isRequired,
+};
+
+const columns = [
+  {
+    title: "Mine Name",
+    width: 300,
+    dataIndex: "mine_name",
+    render: (text, record) => (
+      <div key={record.mine_guid}>
+        <Link to={router.MINE_SUMMARY.dynamicRoute(record.mine_guid)}>{text}</Link>
+      </div>
+    ),
+  },
+  {
+    title: "Last Verified By",
+    width: 150,
+    dataIndex: "verifying_user",
+  },
+  {
+    title: "Last Verified On",
+    width: 150,
+    dataIndex: "formatted_timestamp",
+  },
+];
+
+const transformRowData = (verifiedMinesList) =>
+  verifiedMinesList.map((status) => ({
+    key: status.mine_guid,
+    formatted_timestamp: formatDate(status.verifying_timestamp),
+    ...status,
```

Comment:
If we don't need to pass the other `status` properties onto the transformedRow object, we can pull out the properties that we want to modify and then just pass in the rest:
```javascript
verifiedMinesList.map(({mine_guid: key, verifying_timestamp, ...rest}) => ({
  key,
  formatted_timestamp: formatDate(status.verifying_timestamp),
  ...rest,
})
```
Renaming `mine_guid` during destructuring is entirely optional. I just included it since that's how I would do it.

Reply:
ah i see, ok, avoid having the extra member that could be confusing later. changed.